jekyll/_includes/snippets/ja/runner-1.0-deprecation-brownout.adoc

```diff
@@ -0,0 +1 @@
+CAUTION: **セルフホストランナー バージョン 1.0 は、本日一時的に利用できなくなります。バージョン 1.0 は2023年7月27日に恒久的に終了します。
```

Comment:
Want to confirm that we have added the start time of 12 noon EST here as well, and that we still include the "Update to 1.1" link as in https://github.com/circleci/circleci-docs/pull/8149/files#diff-d9d58d9f69dd8872c8545333b93d112be98133f9ffb0e35bf83cd1df8858f1d0